src/store/onboarding/reducer.ts

```diff
@@ -147,6 +153,10 @@ export function reducer(state: State = defaultState, action: Actions): State {
       return { ...state, isOpen: true };
     case getType(closeModal):
       return { ...state, isOpen: false };
+    case getType(displayConfirm):
+      return { ...state, isOpen: false, isConfirmShown: true };
+    case getType(hideConfirm):
+      return { ...state, isOpen: true, isConfirmShown: false };
```

Comment:
Curious why this confirmation was added to the Redux store?

We've added a couple modals to Redux, so they could be launched from anywhere in the UI. However, the onboarding modal could easily display its own confirmation dialog.

Reply:
Showing two modals at the same time makes the confirmation dialog to be hidden behind the wizard modal. To fix this, I had to hide the wizard modal. If the confirmation modal `isOpen` state isn't stored in Redux then I will have to pass two onClose functions. One for closing the wizard modal and the other to show the confirmation modal.

To me it feels cleaner to put them in one place instead of passing two props that may sound the same.

Reply:
Ok, that sounds reasonable. Thanks for the explanation.